project.py

```diff
@@ -193,17 +190,17 @@ def _jira_type_mapping(self, issue_type):
         if issue_type == 'bug':
             return 'bug'
         if issue_type == 'improvement':
```

Comment:
I need to review that part about issue type retrieved from labels in post process script.